frontend/src/pages/connectionTypes/const.ts

```diff
@@ -0,0 +1,20 @@
+export const connectionTypesPageTitle = 'Connection types';
+export const connectionTypesPageDescription =
+  'Create and manage connection types for users in your organization. Connection types include customizable fields and optional default values to decrease the time required to add connections to data sources and sinks.';
```

Comment:
Seems unnecessary to place these in const instead of inline where they are used only once.